controllers/users/_list_toolbar.htm

```diff
@@ -64,4 +64,19 @@
         </ul>
     </div>
 
+    <?=
+        /**
+         * @event rainlab.user.extendListToolbar
+         * Fires when user list toolbar is rendered.
+         *
+         * Example usage:
+         *
+         *     Event::listen('rainlab.user.extendListToolbar', function ($controller) {
+         *         // return your HTML here
```

Comment:
Could you provide an example of markup that would make sense to include here?

Reply:
Added.